schematic/utils/examples/csv_to_schemaorg_usage.py

```diff
@@ -6,22 +6,17 @@
 from schematic.schemas.explorer import SchemaExplorer
```

Comment:
Could I get clarification on the distinction between this file and `examples/csv_to_schemaorg.py`? They seem redundant. Could this be a leftover from restructuring the repository?

Reply:
If so, I recommend deleting one of the two files (presumably this one since the other was updated with a CLI).

Reply:
Addressed in issue #328 (see comments). We do intend to delete these, since they serve exactly the same purpose.